guides/css.md

```diff
@@ -0,0 +1,29 @@
+# Alternatives to `$.css()`
+
+
+## Option 1:
+
+Use getComputedStyle instead. !Please note `getComputedStyle` triggers a reflow/recalc.
+
+```js
+// get entire CSSStyleDeclaration Object
+getComputedStyle(elem);
+```
+
+```js
+// get a specific style (in this example 'color')
+getComputedStyle(elem)['color'];
+```
+
+---
+
+
+## Option 2:
+
+Use elem#style instead.
+
+```js
+elem.style.borderWidth = '20px';
+```
+
```

Comment:
Think we should suggest `style` as a way to get styles as option 1, since for most cases it works and is less costly.